src/net/types.ts

```diff
@@ -262,6 +267,8 @@ export interface ITransportOptions {
   // contentProtectionParser? : IContentProtectionParser;
 }
 
+export type CustomRepresentationFilter = (representation: Representation) => boolean;
```

Comment:
We add the ``I`` prefix even for ``type``s just to make the difference between a TypeScript type/interface and a real JavaScript object we can use.

Reply:
Also, why Custom there? It's not like we replace an existing one.